.buildkite/pipeline.yml

```
@@ -100,11 +100,14 @@ steps:
   - label: ":argo: Update fleet image tag for fleet-controller using gpctl"
```

looks like a typo here
```suggestion
  - label: ":argo: Update fleet image tag for fleet-server using gpctl"
```